Instruments/bridge12.py

```diff
@@ -482,11 +479,15 @@ def zoom(self, dBm_increment=2, n_freq_steps=15):
         # with the new time constant added for freq_sweep, should we eliminate fast_run?
         rx, tx = self.freq_sweep(freq, fast_run=True)
         # }}}
+        self.freq_bounds = r_[start_f,stop_f]
```

Comment:
this is new -- no explanation of purpose here

Reply:
actually, I read the code for this, and you definitely don't want this here.  These are the outer bounds, and these are determined only once, at low power.